frontend/src/pages/projects/screens/spawner/connections/ConnectionsFormSection.tsx

```diff
@@ -122,26 +128,37 @@ export const ConnectionsFormSection: React.FC<Props> = ({
               variant="secondary"
               isAriaDisabled={unselectedConnections.length === 0}
               onClick={() => setShowAttachConnectionsModal(true)}
-              ref={tooltipRef}
+              ref={connectionsTooltipRef}
             >
               Attach existing connections
             </Button>
             {unselectedConnections.length === 0 && (
               <Tooltip
                 id="no-connections-tooltip"
-                content="No existing connections available"
-                triggerRef={tooltipRef}
+                content="No connections available"
+                triggerRef={connectionsTooltipRef}
               />
             )}
           </FlexItem>
           <FlexItem>
             <Button
               data-testid="create-connection-button"
               variant="secondary"
+              id="no-connection-types-tooltip"
+              content="No connection types available"
+              isAriaDisabled={enabledConnectionTypes.length === 0}
               onClick={() => setManageConnectionModal({ connection: undefined, isEdit: false })}
+              ref={connectionTypesTooltipRef}
```

Comment:
Missing `aria-describedBy` as is done with other tooltips.

Reply:
Nice catch, updated